Testreglar/1.3.3/App/app-1.3.3a.json

```diff
@@ -0,0 +1,426 @@
+{
+	"namn": "App-1.3.3a Instruksjonar for forstå og betene skjema, er ikkje utelukkande avhengig av sensoriske eigenskapar 2023",
+	"id": "app-1.3.3a",
+	"testlabId": 499,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nn",
+	"kravTilSamsvar": "<ul>\r\n<li>Skjemainstruksjon som viser til sensoriske egenskaper (form, størrelse, visuell plassering, orientering eller lyd), har også annen informasjon, som lar brukeren forstå og betjene komponenten eller</li>\r\n<li>I skjemainstruksjon brukes det ord som er vanlige for å vise leserekkefølge, og rekkefølgen er den samme ved sveiping.</li>\r\n</ul>",
```

Comment:
"I skjemainstruksjon brukes det ord som er vanlige for å vise leserekkefølge".   Skjønner ikke hva som er ment her?    Hva er kilden?

Reply:
Hvilken steg i testregelen verifisere dette?

Reply:
Det står tilvarende i eksisterende testregel i steg 3.11 og 3.1.12. 
Kilde forståartikkelen: 
> In some languages, it is commonly understood that "above" refers to the content previous to that point in the content and "below" refers to the content after that point. In such languages, if the content being referenced is in the appropriate place in the reading order and the references are unambiguous, statements such as "choose one of the links below" or "all of the above" would conform to this Success Criterion.